### ISSUE TITLE
CJS build can't be required

### ISSUE THREAD

**Issue by TonyRL** [le0pard/re2js#4]
Repro:
https://runkit.com/64d64e12d266180008287aeb/64d64e125418b5000813fd4a
```js
const { RE2JS } = require('re2js');
const p = RE2JS.compile('abc');
console.log(p.pattern());
console.log(p.flags());
```

## What's expected

```
abc
0
```

## What's actually happened

```
Error: require() of ES Module /app/available_modules/1691766291000/re2js/build/index.cjs.js from /app/index.js not supported.
index.cjs.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename index.cjs.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /app/available_modules/1691766291000/re2js/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

**Comment by le0pard** [comment]
Oh, thanks. Need rename file to have `cjs` instead `js` extension

**Comment by le0pard** [comment]
@TonyRL should be fixed for [0.3.1](https://github.com/le0pard/re2js/releases/tag/0.3.1) version.

```bash
$ yarn node                                                                                                                                                                                                                                                                                           release_76 8d9b2d34e ✗
Welcome to Node.js v20.3.1.
Type ".help" for more information.
> const { RE2JS } = require('re2js');
undefined
> const p = RE2JS.compile('abc');
undefined
> console.log(p.pattern());
abc
undefined
> console.log(p.flags());
0
```

**Comment by TonyRL** [comment]
Thanks for the speedy fixes.
